css/style.css

```diff
@@ -438,9 +443,16 @@ video {
     max-width: 960px;
   }
 
+  .pointer-list {
+    margin: 33px 0px 60px;
+  }
+
+  .pointer-list__oval:last-child {
+    margin-right: 0;
```

Comment:
Why are we writing this property value again? Is it necessary? I see it twice, first one is in the mobile classes.

Reply:
+1

Reply:
you're right, I deleted that thing